bundles/org.openhab.binding.elroconnects/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,268 @@
+			<channel id="lowbattery" typeId="system.low-battery"/>
+		</channels>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="integer" required="true">
+				<label>Device ID</label>
+				<description>ID of the ELRO Connects Device.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<thing-type id="powersocket">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="connector"/>
+		</supported-bridge-type-refs>
+		<label>Motion sensor</label>
+		<description>ELRO Connects power socket</description>
+		<channels>
+			<channel id="powerstate" typeId="system.power"/>
+		</channels>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="integer" required="true">
+				<label>Device ID</label>
+				<description>ID of the ELRO Connects Device.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="scene">
+		<item-type>String</item-type>
+		<label>Scene</label>
+		<description>Scene</description>
+	</channel-type>
+
+	<channel-type id="smokealarm">
+		<kind>trigger</kind>
+		<label>Smoke Alarm</label>
+		<description>Smoke alarm</description>
+		<tags>
+			<tag>Alarm</tag>
+			<tag>Smoke</tag>
+		</tags>
+	</channel-type>
+	<channel-type id="coalarm">
+		<kind>trigger</kind>
+		<label>CO Alarm</label>
+		<description>CO alarm</description>
+		<tags>
+			<tag>Alarm</tag>
+			<tag>CO</tag>
+		</tags>
+	</channel-type>
+	<channel-type id="heatalarm">
+		<kind>trigger</kind>
+		<label>Heat Alarm</label>
+		<description>Heat alarm</description>
+		<tags>
+			<tag>Alarm</tag>
+			<tag>Temperature</tag>
+		</tags>
+	</channel-type>
+	<channel-type id="wateralarm">
+		<kind>trigger</kind>
+		<label>Water Alarm</label>
+		<description>Water alarm</description>
+		<tags>
+			<tag>Alarm</tag>
+			<tag>Water</tag>
+		</tags>
+	</channel-type>
+	<channel-type id="entry">
+		<item-type>Contact</item-type>
+		<label>Entry Contact</label>
+		<description>Door/window contact</description>
+		<tags>
+			<tag>OpenState</tag>
+		</tags>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="mutealarm">
+		<item-type>Switch</item-type>
+		<label>Mute Alarm</label>
+		<description>Mute</description>
+		<category>SoundVolume</category>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
+	</channel-type>
+	<channel-type id="testalarm">
+		<item-type>Switch</item-type>
+		<label>Test Alarm</label>
+		<description>Test alarm</description>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
+	</channel-type>
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Current temperature</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state pattern="%.0f %%" readOnly="true"/>
```

Comment:
```suggestion
		<state readOnly="true" pattern="%.1f %unit%" />
```
You should leave the unit here so it auto adds degrees C or F for the user. Also did you want no decimal places or is 1 useful?

See https://next.openhab.org/docs/concepts/units-of-measurement.html#defining-channeltypes

Reply:
I will put in 1, but I am even not sure the api returns decimals.

Reply:
Consider what happens when the api uses Fahrenheit and the core converts it to celcius, the extra decimal gives less rounding errors. You can change the default display to whatever you wish using metadata on the item. I think it is called pattern.